constants/labels.js

```diff
@@ -0,0 +1,64 @@
+
+const prefixes = {
```

Comment:
I think even though in the `labels.js`, might be good to include `"LABEL"` in the label constant identifiers?